cabal-install/tests/PackageTests/InitConfig/Check.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module PackageTests.InitConfig.Check
```

Comment:
I'm looking at writing the unit tests. To be able to call initConfigAction I need to export it in Main.hs right? I could do that but it feels wrong :P
Is there another way of doing it or what do you think about exporting it?